spec/linter-eslint-spec.js

```diff
@@ -213,6 +215,46 @@ describe('The eslint provider for Linter', () => {
     })
   })
 
+  describe('when a file is not specified in .eslintignore file', async () => {
```

Comment:
What's the point of this entire spec? Isn't it just testing that `copyFileToTempDir` works, and that linting works? (Both of which are tested/used elsewhere).

Reply:
This spec is to explicitly verify that linting occurs if no `.eslintignore` file is found before proceeding to test `eslintIgnore` key. There are already tests for what happens when `.eslintignore` is specified, but nothing to clearly verify what happens in it's absence. I guess you could think of this as a dummy-check for some of the other tests.

I personally did not believe the result from the other 2 specs I wrote until I wrote this one too. I thought I did something wrong until I could see clearly and for sure in a single test that "Yes, if you omit a `.eslintignore` you will get linting".

Reply:
Virtually every other spec (except the rare few that also deal with `.eslintignore` files) handles this.... _but_ they all do so under the state of `atom.config.set('linter-eslint.disableEslintIgnore', true)` which makes a good point for having a fully isolated spec like this, I'd be fine with leaving it in.